assets/base.css

```diff
@@ -74,7 +83,7 @@
 /* base */
 
 .page-width {
-  max-width: var(--page-width);
+  max-width: 130rem;
```

Comment:
instead of adding width directly update the variable that is assigned that width